tests/integration/test_charm.py

```diff
@@ -24,26 +24,43 @@
 log = logging.getLogger(__name__)
 
 METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
+KATIB_CONFIG = "volumes-web-app-viewer-spec-ck6bhh4bdm"
```

Comment:
Katib? Also, I think this name will be different on every deployment.

Reply:
Fixing this